src/functions/assertions/Be.ps1

```diff
@@ -38,6 +38,9 @@ function Should-Be ($ActualValue, $ExpectedValue, [switch] $Negate, [string] $Be
     return [PSCustomObject] @{
         Succeeded      = $succeeded
         FailureMessage = $failureMessage
+        ActualValue    = Format-Nicely $ActualValue
```

Comment:
Agreed, for now the main purpose is for display of the information, but it can be more raw in the future that format-nicely might be cleaning up, my main goal was to just make sure it stringifies easily.